mne/io/eeglab/eeglab.py

```diff
@@ -212,19 +242,39 @@ class RawEEGLAB(_BaseRaw):
         Path to the .set file. If the data is stored in a separate .fdt file,
         it is expected to be in the same folder as the .set file.
     montage : str | None | instance of montage
-        Path or instance of montage containing electrode positions.
-        If None, sensor locations are (0,0,0). See the documentation of
+        Path or instance of montage containing electrode positions. If None,
+        sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
+    eog : list | tuple | 'auto'
+        Names or indices of channels that should be designated EOG channels.
+        If 'auto', the channel names containing ``EOG`` or ``EYE`` are used.
+        Defaults to empty tuple.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated MISC
+        channels. Values should correspond to the electrodes in the .set file.
+        Default is ``()``.
+    event_id : dict | None
+        The ids of the events to consider. If None (default), an empty dict is
+        used and `event_id_func` (see below) is called on every event value.
```

Comment:
Single are supposed to end up being references to variables in the doc, but
I've never seen it work properly. NumPy and SciPy use single, check out the
NumPy docstring standard doc.